packages/desktop-client/src/components/table.tsx

```diff
@@ -796,15 +796,28 @@ export function TableHeader({
   );
 }
 
-export function SelectedItemsButton({ name, items, onSelect }) {
-  const selectedItems = useSelectedItems();
+type SelectedItemsButtonProps<T extends MenuItem = MenuItem> = {
+  name: ((count: number) => string) | string;
+  items: Array<T | typeof Menu.line>;
+  onSelect: (name: string, items: Array<T | typeof Menu.line>) => void;
+};
+
+export function SelectedItemsButton<T extends MenuItem = MenuItem>({
+  name,
+  items,
+  onSelect,
+}: SelectedItemsButtonProps<T>) {
+  const selectedItems = useSelectedItems<T>();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏don't need this anymore
```suggestion
  const selectedItems = useSelectedItems();
```